### ISSUE TITLE
Call to undefined method Grav\Common\Session::getFlashObject()

### ISSUE THREAD

**Issue by tomzx** [getgrav/grav-plugin-form#92]
Form plugin 2.0.2
Grav ~1.0 / getgrav/grav@2709938f9fa7241c312611fea25583739c108344

```php
Whoops\Exception\ErrorException thrown with message "Call to undefined method Grav\Common\Session::getFlashObject()"

Stacktrace:
#8 Whoops\Exception\ErrorException in /<path>/user/plugins/form/form.php:209
#7 Grav\Plugin\FormPlugin:onPagesInitialized in /<path>/vendor/symfony/event-dispatcher/EventDispatcher.php:181
#6 call_user_func:{/<path>/vendor/symfony/event-dispatcher/EventDispatcher.php:181} in /<path>/vendor/symfony/event-dispatcher/EventDispatcher.php:181
#5 Symfony\Component\EventDispatcher\EventDispatcher:doDispatch in /<path>/vendor/symfony/event-dispatcher/EventDispatcher.php:46
#4 Symfony\Component\EventDispatcher\EventDispatcher:dispatch in /<path>/vendor/rockettheme/toolbox/Event/src/EventDispatcher.php:23
#3 RocketTheme\Toolbox\Event\EventDispatcher:dispatch in /<path>/system/src/Grav/Common/Grav.php:407
#2 Grav\Common\Grav:fireEvent in /<path>/system/src/Grav/Common/Grav.php:245
#1 Grav\Common\Grav:process in /<path>/index.php:38
#0 {main} in /<path>/index.php:0
```

**Comment by rhukster** [comment]
Make sure your Grav version is fully up to date.

**Comment by tomzx** [comment]
Is there any system in place that is supposed to alert me if my dependencies are wrong? I see there's

```yaml
dependencies:
    - { name: grav, version: '>=1.1.4' }
```

in the `blueprint.yaml`. Shouldn't this have prevented me from installing the latest form plugin since my grav version is older?

**Comment by rhukster** [comment]
Technically yes, but we found a bug in the admin plugin (that has now been fixed in v1.2.2) that skipped dependency checks if there was more than 1 plugin updated at a time.   

https://github.com/getgrav/grav-plugin-admin/commit/a86623f432cc34363affdb9252f22178cc853219

So next time this wont happen.
